src/alerts/service-alerts-polling.test.ts

```diff
@@ -18,9 +24,10 @@ describe('startPollingServiceAlerts', () => {
   });
 
   it('periodically fetches service alerts and updates store', async () => {
```

Comment:
// Arrange // Act and // Assert test section comments here and in service-alerts.test.ts too please.

Reply:
Done in e7cbbcfb6183d28ff7c80261f9da72d1d493ef8e.